hedera-node/src/main/java/com/hedera/services/contracts/execution/HederaMessageCallProcessor.java

```diff
@@ -56,8 +59,16 @@ public void start(final MessageFrame frame, final OperationTracer operationTrace
         final var hederaPrecompile = hederaPrecompiles.get(frame.getContractAddress());
         if (hederaPrecompile != null) {
             executeHederaPrecompile(hederaPrecompile, frame, operationTracer);
+            ((HederaOperationTracer) operationTracer)
```

Comment:
I think both of these secontions can be compressed into one call after the if/then block, with the same check on CODE_EXECUTING.  So drop this call and pull the call in the else out so both paths execute it.

Reply:
Yup, much cleaner, fixed